_examples/simple/main.go

```diff
@@ -61,24 +62,26 @@ func validateStruct() {
 		// this check is only needed when your code could produce
 		// an invalid value for validation such as interface with nil
 		// value most including myself do not usually have code like this.
-		if _, ok := err.(*validator.InvalidValidationError); ok {
+		if errors.Is(err, &validator.InvalidValidationError{}) {
```

Comment:
I don't think it works

This would be better

```suggestion
		if errors.As(err, &validator.InvalidValidationError{}) {
```

errors.Is is used for sentinel errors.
errors.As is used to check an error is from a certain type, and converts it

Reply:
unfortunately, this introduced "panic: errors: *target must be interface or implement error", because double pointer shopuld be used in this case for for `As()`.
fix in https://github.com/go-playground/validator/pull/1378

i guess this suggestion was accepted by PR author without actual run of code.